### ISSUE TITLE
Changing from BuildWebHost to CreateWebHostBuilder is a really bad idea

### ISSUE THREAD

**Issue by tomecho** [aspnet/Docs#7087]
EF Core is basically what everyone uses on .net core and for ease of use it can pull the db context setup from the BuildWebHost method of the Program.cs, switching to CreateWebHostBuilder makes the user implement a different pattern.
 https://docs.microsoft.com/en-us/ef/core/miscellaneous/cli/dbcontext-creation

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4fa0f562-bdf0-eeea-f076-0c3e05222f75
* Version Independent ID: 8269d917-3b32-1607-71da-c756d4d7ad32
* Content: [Migrate from ASP.NET Core 2.0 to 2.1](https://docs.microsoft.com/en-us/aspnet/core/migration/20_21?view=aspnetcore-2.1)
* Content Source: [aspnetcore/migration/20_21.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/migration/20_21.md)
* Product: **asp.net-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
@tomecho  did you find a problem with the document to report?

**Comment by tomecho** [comment]
Yes, maybe I wasn't clear, https://docs.microsoft.com/en-us/aspnet/core/migration/20_21?view=aspnetcore-2.1#changes-to-main making these changes to the BuildWebHost method is not recommended by EF Core.  Basically its a ease of use concern.

**Comment by Rick-Anderson** [comment]
@tomecho  writes
> making these changes to the BuildWebHost method is not recommended by EF Core. 

Where did you get that information?

**Comment by tomecho** [comment]
https://docs.microsoft.com/en-us/aspnet/core/migration/1x-to-2x/?view=aspnetcore-2.1#update-main-method-in-programcs

**Comment by tomecho** [comment]
https://github.com/aspnet/EntityFrameworkCore/issues/12331

**Comment by Rick-Anderson** [comment]
@tomecho   That is not a recommendation to use `BuildWebHost`
The templates 2.1 templates the EF team created use `CreateWebHostBuilder`

`dotnet new webapp`

**Comment by seanmars** [comment]
How can i use SeedData in 2.1?
